src/core.c/Promise.rakumod

```diff
@@ -296,11 +322,35 @@ my class Promise does Awaitable {
             my $then-p := self.new(:$!scheduler);
             my $vow := $then-p.vow;
             self!PLANNED-THEN( $then-p,
-                               $vow,
-                               { $!status == Kept
-                                   ?? $vow.keep($!result)
-                                   !! do { my $*PROMISE := $then-p; $vow.keep(code(self)) } },
-                               $synchronous)
+                    $vow,
+                    { $!status == Kept
+                            ?? $vow.keep($!result)
+                            !! do { my $*PROMISE := $then-p; $vow.keep(code(self)) } },
+                    $synchronous)
+        }
+    }
+    multi method orelse(Promise:D: &code, :$synchronous) is revision-gated('6.e') {
```

Comment:
The `revision-gated` argument here doesn't match the format of the other ones in your PR.  Ironically, I think this one is the only **correct** one -- the other ones need to be changed to have the `.` in them.